### ISSUE TITLE
Create a dark theme 

### ISSUE THREAD

**Issue by ExpressHermes** [mann27/News24x7-Client#45]

**Comment by rohan2734** [comment]
@mann27  i would like to work on this issue.

**Comment by mann27** [comment]
Sure! @ExpressHermes will describe it soon to you. For taking up tasks or in case of doubts tag both of us. So that whoever sees it first can respond.

**Comment by ExpressHermes** [comment]
@rohan2734 Do you have any doubts regarding this issue?

**Comment by rohan2734** [comment]
@mann27  @ExpressHermes  not yet, like i need to try the color combination for dark mode right?

**Comment by ExpressHermes** [comment]
Yes just try different combinations and if you like any, show us before working on it.

**Comment by rohan2734** [comment]
> Yes just try different combinations and if you like any, show us before working on it.

ok sure.

**Comment by mann27** [comment]
@rohan2734 any updates on it?

**Comment by rohan2734** [comment]
@mann27  i am not working on it right. I already said before long back.

**Comment by mann27** [comment]
Cool!!
